portfolio/src/main/webapp/about.js

```diff
@@ -1,104 +1,105 @@
+ * Advances the game to the next question. Loads in the first question
+ * game has not yet started. Otherwise, checks answer against user
  * selected anwer and alerts if a lie or truth has been chosen.
  */
-function advanceGame()  {
-    let chosenAnswer = -1;
-    const advanceGameButton = document.getElementById("advance-game-button");
-    if (advanceGameButton.innerHTML === "Play Two Truths One Lie") {
-        advanceGameButton.innerHTML = "Next";
+function advanceGame() {
+  let chosenAnswer = -1;
+  const advanceGameButton = document.getElementById('advance-game-button');
+  if (advanceGameButton.innerHTML === 'Play Two Truths One Lie') {
+    advanceGameButton.innerHTML = 'Next';
+  }
+  const chosenRadios =
+      document.querySelectorAll('input[name="selected-answer"]');
+  for (chosenRadio of chosenRadios) {
+    if (chosenRadio.checked) {
+      chosenAnswer = chosenRadio.value;
     }
-    const chosenRadios = document.querySelectorAll('input[name="selected-answer"]');
-    for (chosenRadio of chosenRadios) {
-        if (chosenRadio.checked) {
-            chosenAnswer = chosenRadio.value;
-        }
-    }``
-    if (chosenAnswer != -1) {
-        if (correctAnswer == chosenAnswer) {
-            alert('This is the lie!');
-        } else {
-            alert('This is the truth!');
-        }
+  }
+  ``
+  if (chosenAnswer != -1) {
+    if (correctAnswer == chosenAnswer) {
+      alert('This is the lie!');
+    } else {
+      alert('This is the truth!');
     }
-    const truthIndices = loadInTruth();
-    correctAnswer = loadInLie(truthIndices);
+  }
+  const truthIndices = loadInTruth();
+  correctAnswer = loadInLie(truthIndices);
 }
 
 /**
  * Populates the page with two true facts in randomized indexes
- * 
+ *
  * @return {Array} an array populated with two numbers containing
  * locations of the true facts
  */
 function loadInTruth() {
+  let firstTruthValue = Math.floor(Math.random() * truths.length);
+  let secondTruthValue = Math.floor(Math.random() * truths.length);
+  while (firstTruthValue === secondTruthValue) {
+    secondTruthValue = Math.floor(Math.random() * truths.length);
+  }
 
-    let firstTruthValue = Math.floor(Math.random() * truths.length);
-    let secondTruthValue = Math.floor(Math.random() * truths.length);
-    while (firstTruthValue === secondTruthValue) {
-        secondTruthValue = Math.floor(Math.random() * truths.length);
-    }
-
-    let firstTruthElementIndex = Math.floor(Math.random() * 3);
-    let secondTruthElementIndex = Math.floor(Math.random() * 3);
-    while (firstTruthElementIndex === secondTruthElementIndex) {
-        secondTruthElementIndex = Math.floor(Math.random() * 3);
-    }
+  let firstTruthElementIndex = Math.floor(Math.random() * 3);
+  let secondTruthElementIndex = Math.floor(Math.random() * 3);
+  while (firstTruthElementIndex === secondTruthElementIndex) {
+    secondTruthElementIndex = Math.floor(Math.random() * 3);
+  }
```

Comment:
Please remember to format all your code before merging with master so that these types of edits aren't being made in other PRs.

Reply:
Will use clang-format-path from now on